src/net.cpp

```diff
@@ -2340,6 +2340,27 @@ void CConnman::ProcessFixedSeeds(std::chrono::microseconds start)
                 seed_addrs.erase(std::remove_if(seed_addrs.begin(), seed_addrs.end(),
                                                 [&fixed_seed_networks](const CAddress& addr) { return fixed_seed_networks.count(addr.GetNetwork()) == 0; }),
                                  seed_addrs.end());
+                Shuffle(seed_addrs.begin(), seed_addrs.end(), FastRandomContext());
+
+                // Make AddrFetch connections to fixed seeds first. This reduces the
+                // load on the fixed seeds that would otherwise be serving blocks for
+                // many new peers requiring IBD. Try this with multiple fixed seeds for
+                // a better diversity of received addrs and because some may be offline.
+                LogPrintf("Initiating AddrFetch connections to fixed seeds. This might take up to 2 minutes.\n");
```

Comment:
In "net: Make AddrFetch connections to fixed seeds" dfd635bcc9b3f3615cabe115af302df33efba6a1: Perhaps it would be interesting to specify in the log it will initiate 10 AddrFetch connections instead of just saying "to fixed seeds".

Reply:
added the 10